yarn.lock

```diff
@@ -6397,7 +6394,7 @@ react-dom@^16.4.2:
   version "2.1.5"
   resolved "https://codeload.github.com/matrix-org/react-gemini-scrollbar/tar.gz/f64452388011d37d8a4427ba769153c30700ab8c"
   dependencies:
-    gemini-scrollbar matrix-org/gemini-scrollbar#91e1e566
+    gemini-scrollbar matrix-org/gemini-scrollbar#b302279
```

Comment:
Hmm, something seems wrong here... matrix-org/react-gemini-scrollbar#f644523 depends on matrix-org/gemini-scrollbar#91e1e566, so I don't think this part of the lock file should have changed... Maybe try deleting this single block form `yarn.lock` and re-running `yarn install`?

Reply:
Indeed this is funky

Reply:
that seems to have worked

Reply:
That tends to be the best procedure to get `yarn` to do sensible things when tweaking a specific package, even though it feels like it should have to be that way... 😅